docs/en/tutorials/embeddables.rst

```diff
@@ -77,6 +77,59 @@ the embedded object.
         $this->address = new Address();
     }
 
+Nullable Embeddables Hydration
+------------------------------
+In case all fields in the embeddable are null value, the entity has a empty
+embeddatable object is hydrated. Properties in nullable embeddable should have
+default value. You can normalize entities on ``postLoad`` event
+(in Doctrine ORM lifecycle events), to avoid getting the empty embeddable object
+instead of a null value.
+
+.. configuration-block::
+
+    .. code-block:: attribute
+
+        <?php
+
+        #[Entity]
```

Comment:
Please import the Attributes in this code examples.

Reply:
fixed 5b6bd6f4bfb2aa3241a97f8115bd6fb1df88db85

@SenseException 
Is this right?
Would it be good to fix other examples in this file?